README.md

```diff
@@ -207,3 +196,40 @@ gradle uploadArchives
 
 Currently all builds are considered snapshots.  The archive name is based off of `git describe`.
 
+###How to contribute
+(Note: section inspired by, and some text copied from, [Apache Parquet](https://github.com/apache/parquet-mr))
+ 
+We welcome all contributions to the GATK project. The contribution can be a [issue report]( https://github.com/broadinstitute/gatk/issues) or a [pull request](https://github.com/broadinstitute/gatk/pulls). If you're not a commiter, you will need to [make a fork](https://help.github.com/articles/fork-a-repo/) of the gatk repository and [issue a pull request](https://help.github.com/articles/be-social/) from your fork.
```

Comment:
Should define who is eligible to be a committer / how does one become a committer, and also the alternate option (external contributor?)